common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
   "message.voicechat.group": "Group",
-  "message.voicechat.voice_chat_volume": "Voice chat volume: %s",
-  "message.voicechat.microphone_amplification": "Microphone amplification: %s",
-  "message.voicechat.mic_test_unavailable": "Microphone testing unavailable",
-  "message.voicechat.mic_test.disabled": "Press to enable microphone testing",
-  "message.voicechat.mic_test.enabled": "Press to disable microphone testing",
+  "message.voicechat.voice_chat_volume": "Voice Chat Volume: %s",
+  "message.voicechat.microphone_amplification": "Microphone Amplification: %s",
+  "message.voicechat.mic_test_unavailable": "Microphone testing unavailable.",
+  "message.voicechat.mic_test.disabled": "Press to enable microphone testing.",
+  "message.voicechat.mic_test.enabled": "Press to disable microphone testing.",
   "message.voicechat.voice_chat_unavailable": "Voice chat unavailable",
   "message.voicechat.voice_chat_not_connected": "Voice chat not connected",
-  "message.voicechat.activation_type": "Activation type: %s",
-  "message.voicechat.activation_type.ptt": "Push to talk",
+  "message.voicechat.activation_type": "Activation Method: %s",
+  "message.voicechat.activation_type.ptt": "Push to Talk",
   "message.voicechat.activation_type.voice": "Voice",
-  "message.voicechat.voice_activation": "Voice activation threshold: %s dB",
+  "message.voicechat.voice_activation": "Voice Activation Threshold: %s dB",
   "message.voicechat.voice_activation.disabled": "0 dB deactivates the microphone!",
-  "message.voicechat.adjust_volumes": "Adjust volumes",
+  "message.voicechat.adjust_volumes": "Adjust Volumes",
   "message.voicechat.volume_amplification": "Amplification: %s",
   "message.voicechat.muted": "Muted",
-  "message.voicechat.adjust_volume_player": "Adjust volume of %s",
-  "message.voicechat.no_microphone": "No microphone available",
-  "message.voicechat.no_speaker": "No speaker available",
-  "message.voicechat.select": "Select",
-  "message.voicechat.select_microphone": "Select microphone",
-  "message.voicechat.select_speaker": "Select speaker",
+  "message.voicechat.no_microphone": "No microphones available",
+  "message.voicechat.no_speaker": "No speakers available",
+  "message.voicechat.select_microphone": "Select a Microphone",
+  "message.voicechat.select_speaker": "Select Speakers",
   "message.voicechat.back": "Back",
   "message.voicechat.client_not_connected": "Client not connected",
   "message.voicechat.failed_to_send_ping": "Failed to send ping: %s",
   "message.voicechat.sending_ping": "Sending ping...",
   "message.voicechat.ping_sent_waiting": "Ping sent. Waiting for response...",
-  "message.voicechat.ping_received": "Got a response in %sms",
-  "message.voicechat.ping_received_attempt": "Got a response after %s attempts in %sms",
+  "message.voicechat.ping_received": "Got a response after %s ms",
+  "message.voicechat.ping_received_attempt": "Got a response after %s attempts and %s ms",
```

Comment:
Could you please explain how this works?

When I tested this functionality some time ago (both in the game and using your Rust-based CLI tool), I noticed that if the first few ping packets timed out, the first one that arrived successfully showed a higher delay in ms than usual.

Was it just a coincidence? Is this value not cumulative and only shows the delay of that one packet that finally made it through?

Reply:
The pinging tool sends a packet every second (each ping gets a separate unique ID). Once  you get a ping back it log s the time between the sending and receiving of the ping. So if you would have a ping of over a second, you would get the message of sending the second ping and shortly after that a message of receiving the answer of the first ping. The ping is always in relation to it getting sent and not the pining process getting initiated in general.

Reply:
Thank you for the detailed explanation.

In the meantime, I've changed both of these chat messages back to what they were before.

I have one more question though. If you were to put yourself in the shoes of a user unfamiliar with the topic, seeing the following text for the first time, how would you interpret it?

> Got a response after 3 attempts in 150 ms

To me, depending on how it's read, it may suggest that it was the 3 attempts that took 150 ms, and only the fourth attempt was successful without mentioning how long the packet took. If I were to visualize it, it would look something like this: "Got a response (after 3 attempts in 150 ms)". But I'm sure that the intended meaning is more like this: "Got a response (after 3 attempts) in 150 ms", meaning that the response took 150 ms, with 3 unsuccessful attempts beforehand.

My proposed solution to this problem (**if there is a problem at all**, which is why I'd like to hear your opinion on this) would be to swap the time and the number of attempts. The result would look like this: "Got a response in %s ms after %s attempts".

What do you think? How would you interpret this message?

Reply:
Yeah `Got a response in %s ms after %s attempts` is way better, you are right. The issue is, that we need to switch the parameters in the code, which causes all other translations to be incorrect...

Reply:
From my perspective, swapping the order of the two arguments in the `Component#translatable()` method is easy and straightforward, and I could do it in no time.

If you're worried that this change may cause a lot of extra and potentially unnecessary work for translators, it's actually not as bad as I initially thought after reading your comment. Only 7 out of the 24 translations (Korean, Portuguese, Russian, Swedish, Ukrainian, Vietnamese, and Traditional Chinese) are 100% translated at the moment, and they're the ones that would be noticeably affected by this change. The rest need additional work from translators anyway, so having one more line to translate alongside everything else shouldn't be a problem at all.

What do you think?

Reply:
Yeah, we should change that is a different PR, to avoid confusion, what do you think?